server/app.js

```diff
@@ -13,7 +14,8 @@ app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 
 app.use('/', express.static('build'));
-app.use('/*', express.static('build'));
+app.use('/api', routes);
```

Comment:
Could someone explain this modification to me?

Reply:
@bill-feng Sorry for the confusion, we mistakenly commented it out. I guess the morning lecture by Brent explains.:)